### PR TITLE
Fix a bunch of PINN benchmarks v2

### PR DESCRIPTION
**Pull request by KirillZubov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Static outputs in pdf, markdown, and html reside in [SciMLBenchmarksOutput](http
   - [DifferentialEquations.jl vs Mujuco and DiffTaichi](https://arxiv.org/abs/2012.06684)
   - [DiffEqFlux.jl / DifferentialEquations.jl vs Jax on an epidemic model](https://gist.github.com/ChrisRackauckas/62a063f23cccf3a55a4ac9f6e497739a)
   - [DifferentialEquations.jl vs SciPy vs NumbaLSODA on a stiff ODE](https://gist.github.com/ChrisRackauckas/fd62e005c4c86520306338b6bdae6b79)
+  - [DifferentialEquations.jl vs SciPy vs NumbaLSODA](https://github.com/Nicholaswogan/NumbaLSODA/tree/main/benchmark)
```
Comment (ChrisRackauckas) [inline README.md:24]: ?

File: benchmarks/DAE/ChemicalAkzoNobel.jmd [hunk 2]
```diff
@@ -45,17 +45,16 @@ eqs = [
...
 ModelingToolkit.@named sys = ModelingToolkit.ODESystem(eqs)
 
 tspan = (0.0, 180.0)
-simpsys = structural_simplify(sys)
-mmprob = ODEProblem(simpsys, [], tspan)
+mmprob = ODEProblem(sys, [], tspan)
 sol = solve(mmprob, Rodas4(),abstol=1/10^14,reltol=1/10^14)
 
-odaeprob = ODAEProblem(simpsys,[],tspan)
+odaeprob = ODAEProblem(structural_simplify(sys),[],tspan)
```
Comment (ChrisRackauckas) [inline benchmarks/DAE/ChemicalAkzoNobel.jmd:57]: accidental commits?

Comment (KirillZubov) [reply benchmarks/DAE/ChemicalAkzoNobel.jmd:57]: it is a merge in SciML:pinn

Comment (ChrisRackauckas) [reply benchmarks/DAE/ChemicalAkzoNobel.jmd:57]: But these commits shouldn't be added to that branch?

Comment (ChrisRackauckas) [reply benchmarks/DAE/ChemicalAkzoNobel.jmd]: I just did a rebase to eliminate those.

Comment (KirillZubov) [reply benchmarks/DAE/ChemicalAkzoNobel.jmd]: thanks, yes it looks better now

**Comment by KirillZubov** [comment]
@ChrisRackauckas  how about removing these strategies for all scripts except one? it spends most of time.
```julia
strategies = [NeuralPDE.QuadratureTraining(quadrature_alg = CubaCuhre(), reltol = 1, abstol = 1e-4, maxiters = 100),
              NeuralPDE.QuadratureTraining(quadrature_alg = HCubatureJL(), reltol = 1, abstol = 1e-4, maxiters = 100, batch = 0),
              NeuralPDE.QuadratureTraining(quadrature_alg = CubatureJLh(), reltol = 1, abstol = 1e-4, maxiters = 100),
```

**Comment by KirillZubov** [comment]
@ChrisRackauckas
`ERROR: LoadError: ArgumentError: Path to conda environment is not valid: /cache/julia-buildkite-plugin/depots/5b300254-1738-4989-ae0a-f4d2d937f953/conda/3`

**Comment by ChrisRackauckas** [comment]
Yeah that's odd. @staticfloat did addresses change or something?

**Comment by ChrisRackauckas** [comment]
merged so I could rebase it for the Conda.jl fix.

**Pull request merged**